components/director/internal/domain/document/fixtures_test.go

```diff
@@ -154,3 +156,23 @@ func timeToTimestampPtr(time time.Time) *graphql.Timestamp {
 	t := graphql.Timestamp(time)
 	return &t
 }
+
+func fixUpdateTenantIsolationSubquery() string {
+	return regexp.QuoteMeta(`tenant_id IN ( with recursive children AS (SELECT t1.id, t1.parent FROM business_tenant_mappings t1 WHERE id = ? UNION ALL SELECT t2.id, t2.parent FROM business_tenant_mappings t2 INNER JOIN children t on t.id = t2.parent) SELECT id from children )`)
+}
+
+func fixTenantIsolationSubquery() string {
+	return fixTenantIsolationSubqueryWithArg(1)
+}
+
+func fixUnescapedTenantIsolationSubquery() string {
```

Comment:
I thought about removing them but decided that those functions are small enough (and hard enough to figure out without context) to leave them as is. Furthermore, this tenant isolation query will not change with adding more and more tenant types, therefore it is probably useful to have those utility fixtures when writing new tests in any repository and don't think about it.

If you think it is better to remove them and leave only the used ones I will do so, however, I think it will be easier for further repository tests which potentially can have those requirements.